quickwit/quickwit-metastore/src/metastore/postgresql_metastore.rs

```diff
@@ -172,10 +172,32 @@ async fn mark_splits_as_published_helper(
     .map(|row| row.get(0))
     .fetch_all(tx)
     .await?;
+
     Ok(published_split_ids)
 }
 
-/// Marks mutiple splits for deletion.
+#[instrument(skip(tx))]
+/// Updates the given index's `update_timestamp` to the current
+/// timestamp.
+async fn update_index_last_update_ts(
```

Comment:
`touch_index`?

Reply:
I think `touch_index` is a bit vague about what it actually does, but I can't think of a better name for it :/

Reply:
Then, `update_index_update_timestamp` or `update_index_update_ts` is the most accurate.